src/librustc/dep_graph/dep_node.rs

```diff
@@ -521,12 +521,23 @@ define_dep_nodes!( <'tcx>
     [] ItemAttrs(DefId),
     [] FnArgNames(DefId),
     [] DylibDepFormats(DefId),
-    [] IsAllocator(DefId),
     [] IsPanicRuntime(DefId),
     [] IsCompilerBuiltins(DefId),
     [] HasGlobalAllocator(DefId),
     [] ExternCrate(DefId),
     [] LintLevels,
+    [] IsSanitizerRuntime(DefId),
+    [] IsProfilerRuntime(DefId),
+    [] GetPanicStrategy(DefId),
+    [] IsNoBuiltins(DefId),
+    [] ImplDefaultness(DefId),
+    [] ExportedSymbols(DefId),
+    [] NativeLibraries(DefId),
+    [] PluginRegistrarFn(DefId),
+    [] DeriveRegistrarFn(DefId),
+    [] ForeignCrateDisambiguator(DefId),
+    [] CrateHash(DefId),
+    [] ForeignOriginalCrateName(DefId),
```

Comment:
@nikomatsakis @michaelwoerister Is there not a better way to deal with this?

Reply:
The problem being the duplication? I'd like to combine this macro with the query macro, I'm not sure if something is blocking that.